src/__tests__/myprofile/myprofile.test.js

```diff
@@ -0,0 +1,7 @@
+import Mission from '../../components/missions/mission';
+import MyProfile from "../../components/myprofile/MyProfile"
+import renderWthRedux from '../../utils/testHelpers';
+
+it('should MissionContainer render ', () => {
+           let {getByTestId}  =renderWthRedux(<MyProfile/>)
+});
```

Comment:
[OPTIONAL] 
Would be nice to have a cleaner test without logs and errors.

<img width="687" alt="cr" src="https://user-images.githubusercontent.com/83017311/162196036-60b65420-dfdb-44fd-a449-5c4b0112ed84.png">